### ISSUE TITLE
Conflicting dependencies between Signifypy and KERIA

### ISSUE THREAD

**Issue by nkongsuwan** [WebOfTrust/signifypy#89]
I get the following error when I installed Signifypy using `pip install -r requirements.txt`

```
INFO: pip is looking at multiple versions of keria to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install -r requirements.txt (line 10) and keria==0.0.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    signifypy 0.0.1 depends on keri 1.0.0 (from git+https://****@github.com/weboftrust/keripy.git@development)
    keria 0.0.1 depends on keri 1.0.0 (from git+https://****@github.com/WebOfTrust/keripy.git)

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

It seems to work when I change a line in the `setup.py` so that it is consistent with KERIA.

**Comment by m00sey** [comment]
I was able to update my local copy and install fine. 
I was also able to clone a new copy and install fine.

```
10:07:25 (venv) m00sey@airstrip-1 signifypy ±|development|→ pip install -r requirements.txt
Looking in indexes: https://pypi.org/simple/
Obtaining file:///home/m00sey/dev/testy/signifypy (from -r requirements.txt (line 10))
...
Installing collected packages: wcwidth, pysodium, netifaces, lmdb, hjson, blake3, urllib3, typing-extensions, six, rpds-py, PyYaml, pycparser, PrettyTable, packaging, ordered-set, multidict, multicommand, msgpack, mnemonic, keria, idna, falcon, charset-normalizer, certifi, cbor2, attrs, requests, referencing, http-sfv, hio, cffi, apispec, sseclient, jsonschema-specifications, cryptography, jsonschema, keri, signifypy
Successfully installed PrettyTable-3.9.0 PyYaml-6.0.1 apispec-6.3.0 attrs-23.1.0 blake3-0.3.3 cbor2-5.5.0 certifi-2023.7.22 cffi-1.16.0 charset-normalizer-3.3.1 cryptography-41.0.5 falcon-3.1.1 hio-0.6.9 hjson-3.1.0 http-sfv-0.9.8 idna-3.4 jsonschema-4.19.1 jsonschema-specifications-2023.7.1 keri-1.0.0 keria-0.0.1 lmdb-1.4.1 mnemonic-0.20 msgpack-1.0.7 multicommand-1.0.0 multidict-6.0.4 netifaces-0.11.0 ordered-set-4.1.0 packaging-23.2 pycparser-2.21 pysodium-0.7.14 referencing-0.30.2 requests-2.31.0 rpds-py-0.10.6 signifypy-0.0.1 six-1.16.0 sseclient-0.0.27 typing-extensions-4.8.0 urllib3-2.0.7 wcwidth-0.2.8

[notice] A new release of pip is available: 23.2.1 -> 23.3.1
[notice] To update, run: pip install --upgrade pip
10:07:46 (venv) m00sey@airstrip-1 signifypy ±|development|→
```

**Comment by m00sey** [comment]
@nkongsuwan I opened #90 to remove the dependency.
